### PR TITLE
media-types: show relations between media types

### PR DESCRIPTION
**Pull request by s-urbaniak**
Please ignore this PR if you think it is not useful, it is mere manifestation of my notes and the relations between all the JSON files in this spec.

I am still unsure of the combined output format, see https://github.com/opencontainers/image-spec/issues/54.

### CODE REVIEW

**Comment by wking** [comment]
I don't think specs for
application/vnd.oci.image.serialization.config.v1+json have landed yet
(it seems to be based on discussion in #23), but I'd expect it will
include everything up through
application/vnd.oci.image.manifest.v1+json and possibly
application/vnd.oci.image.manifest.list.v1+json.

It will likely go beyond that, and include several images [1], and
possibly signatures asserting names for those images, but I don't
think there is a consensus yet on how that information will be
structured.

[1]: https://github.com/opencontainers/image-spec/issues/23#issuecomment-212751623

File: Makefile [hunk 1]
```diff
@@ -58,3 +57,7 @@ oci-validate-json: validate.go
 oci-validate-examples: cmd/oci-validate-examples/main.go
 	go build ./cmd/oci-validate-examples
 
+%.dot.png: %.dot
```
Comment (vbatts) [inline Makefile]: I like make patterns, but nothing is depending on the `media-types.dot.png` target

Comment (s-urbaniak) [inline Makefile]: good catch, thanks, also it doesn't appear in the generated doc, will fix.

**Comment by philips** [comment]
@s-urbaniak serialization.rootfs and serialization.config are siblings of image.manifest. Otherwise I think this is helpful for folks and we should probably add language around it.

**Comment by philips** [comment]
@s-urbaniak you have two copies checked in now. Can you remove the outdated one?

**Comment by s-urbaniak** [comment]
@philips fixed, thanks!

**Comment by philips** [comment]
LGTM

@vbatts thoughts?

**Comment by vbatts** [comment]
@s-urbaniak fixed my concerns. LGTM

**Pull request merged**